index.html

```diff
@@ -5,7 +5,94 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.3/css/all.css"
+      integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk"
+      crossorigin="anonymous"
+    />
     <title>Quizzzle</title>
   </head>
-  <body></body>
+  <body>
+    <nav>
+      <dl class="menu">
+        <dt>home</dt>
+        <dd class="active"><a href="#"></a><i class="fas fa-home"></i></a></dd>
```

Comment:
Please consider an `ul` with `li`s  instead. Also: the anchor closes twice in this line. And it needs some text for accessibility.

Reply:
thats why I did the dl so there would be a title for the screen reader. How could I do it correctly?

Reply:
The text needs to be in the anchor. Do it like we did it together in my example.
```html
<nav>
  <ul>
    <li><a href="#">Home</a></li>
   <!-- ... -->
 </ul>
</nav>
```